test/integration/admin_modify_user_data_test.rb

```diff
@@ -16,8 +16,8 @@ class AdminModifyUserDataTest < ActionDispatch::IntegrationTest
 
     assert_equal edit_user_path(admin), current_path
 
+    select('New York', :from => 'user_state')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.